### PR TITLE
[shuffle] Polish tutorials

### PR DESCRIPTION
**Pull request by sunmilee**
## Motivation

Making changes to the shuffle tutorials based on dry run.

### CODE REVIEW

File: shuffle/cli/tutorials/HelloNFT.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Let's try calling the `mint_nft` script function within the console.
 
 > let txn = await helpers.invokeScriptFunction(scriptFunction, typeArguments, args);
 
-> txn = await devapi.waitForTransaction(txn.hash);
+> txn = await devapi.waitForTransactionCompletion(txn.hash);
```
Comment (dimroc) [inline shuffle/cli/tutorials/HelloNFT.md:109]: In a fast follow, let's rename this method to `waitForTransaction` to have parity with the Dev API's `client.ts`:

https://github.com/diem/diem/blob/cb5ada96dd881864ffe0eee0f5dbdca12845eac8/shuffle/move/examples/main/client.ts#L30-L33

**Review by dimroc — DISMISSED** [review]
/land

File: shuffle/cli/README.md [hunk 2]
```diff
@@ -5,6 +5,13 @@ blockchain.
 
 ## Setup
 
+Prerequisites:
+
+```
+Install brew: https://brew.sh/
```
Comment (xli) [inline shuffle/cli/README.md:11]: I think this is not required.
User just need to follow installation doc for deno

**Review by dimroc — APPROVED** [review]
/land

**Pull request closed without merging**